api/utils.go

```diff
@@ -0,0 +1,45 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+
+	"go.sia.tech/jape"
+	"go.sia.tech/renterd/internal/prometheus"
+)
+
+func WriteResponse(jc jape.Context, code int, resp any) error {
```

Comment:
@ChrisSchinnerl do you think it's worth moving this into the `jape` package?

Reply:
The checker can't work with that right now so I'd keep it here for now.